libcodechecker/libclient/product_helper.py

```diff
@@ -93,6 +93,10 @@ def getProducts(self, product_endpoint_filter, product_name_filter):
     def getCurrentProduct(self):
         pass
 
+    @ThriftClientCall
+    def getProductConfiguration(self, product_id):
```

Comment:
Why did you add this to the product helper. Do you want to use it in the client?